data/json/professions.json

```diff
@@ -1593,6 +1607,7 @@
     "type": "profession",
     "id": "rescuer",
     "name": { "male": "Rescuer", "female": "Rescuer" },
+    "requirement": "achievement_kill_1000_monsters",
```

Comment:
Shouldn't this rather be tied to finding the mi-go camp?

Reply:
changing and reuploading